src/conversion/shared.js

```diff
@@ -0,0 +1,33 @@
+function createMap(values, transform, unit) {
```

Comment:
Changing this signature to `function createMap(values, transform = ((i) => i), unit)`, passing default value of the function will simplify a lot of code elsewhere probably.

Reply:
Good idea! Maybe `function createMap(values, unit, transform = ((i) => i))` so we can just drop the last parameter instead of having undefined as the second param

Reply:
Exactly what I mean't.